apps/next-no-cache/sanity.config.ts

```diff
@@ -5,9 +5,9 @@
 
 import {visionTool} from '@sanity/vision'
 import {defineConfig} from 'sanity'
+import {presentationTool} from 'sanity/presentation'
```

Comment:
Thanks to the `link-workspace-packages = deep` in the root `.npmrc` we can use e2e tests that import from `sanity/presentation` and still have it use the workspace version instead of npm 🥳